daemon/web/pages/Dashboard/index.js

```diff
@@ -32,18 +32,21 @@ const styles = {
 
 class DashboardWrapper extends React.Component {
   componentWillMount() {
-    this.props.handleGetProjectDetails();
-    this.props.handleGetContainers();
+    const { handleGetProjectDetails, handleGetContainers } = this.props;
```

Comment:
Not sure why this is necessary - we're adding two extra assignments?

Reply:
The new eslint rules are more strict about destructuring, and I think occasionally having situations like this is a small price to pay to avoid this.props.thing.another.thing that’s already a bit prevalent